webui/apps/website/lib/dtos.ts

```diff
@@ -5,11 +5,6 @@
 /* Do not modify it by hand - just update the pydantic models and then re-run the script
 */
 
-export interface Choice {
```

Comment:
The generator was doing something weird and creating another interface called `Choice1` below.